src/client/pages/Admin/Admin.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { useEffect } from 'react'
+import { useNavigate } from 'react-router-dom'
+
+import { ClientRoutes } from '@meta/app'
+import { Users } from '@meta/user'
+
+import { useUser } from '@client/store/user'
+
+const Admin: React.FC = () => {
+  const user = useUser()
+  const navigate = useNavigate()
+
+  useEffect(() => {
+    if (!Users.isAdministrator(user)) navigate(ClientRoutes.Root.path, { replace: true })
+  }, [navigate, user])
+
+  if (!user) return null
+
+  return <div className="app-view__content">Placeholder</div>
```

Comment:
if there'a user logged in, and it's not admin, on first mount this goes through. if there's a way to avoid this, avoid it.

Reply:
ok, there is no other chance than put a condition around the route

Reply:
`|| !Users.isAdministrator(user)`